src/zerocoin_v3.cpp

```diff
@@ -965,10 +1043,10 @@ int CSigmaState::GetCoinSetForSpend(
 
 std::pair<int, int> CSigmaState::GetMintedCoinHeightAndId(
         const sigma::PublicCoin& pubCoin) {
-    auto coinIt = mintedPubCoins.find(pubCoin);
+    auto coinIt = containers.GetMints().find(pubCoin);
```

Comment:
Why not write auto coinIt = GetMints().find(pubCoin); here? more readable I think.